handler.fish

```diff
@@ -0,0 +1,21 @@
+function __fish_command_not_found_on_interactive --on-event fish_prompt
+    functions --erase __fish_command_not_found_handler
```

Comment:
In the Bash/Zsh script we override the command-not-found handler if there’s already one.